demos/button.html

```diff
@@ -189,5 +189,26 @@
       </section>
     </main>
     <script src="assets/material-components-web.js" charset="utf-8"></script>
+    <script>
+      // Because we load our CSS via webpack, we need to ensure that all of the correct styles
```

Comment:
LGTM, but this is really unfortunate :-/ We may want to reconsider our demo infrastructure at some point, as the current one makes it hard to demonstrate server-side rendering / correct first renders, not to mention that it makes things much harder to understand for folks that prefer to look at demos rather than readmes.